### PR TITLE
Updating home content

### PR DESCRIPTION
**Pull request by manecipoglu**
|                        Web Dev Path                      |
| :-----------------------------------------------------------: |
| [**45**](https://github.com/MarianaSouza/web-dev-path/issues/45) |

#### What is this change?
Update the home content with  two headers one button and 4 card elements with responsive styling. 

#### Were there any complications while making this change?
Chaging of a media break point from 600px to 728px.

#### How did you verify this change?
Locally.

#### When should this be merged?
After code review.

### CODE REVIEW

File: styles/Home.module.css [hunk 1]
```diff
@@ -30,55 +30,81 @@ hr.divider {
...
+}
+
 .grid {
   display: flex;
   align-items: center;
   justify-content: center;
   flex-wrap: wrap;
-
-  margin-top: 3rem;
+  /* max-width: 50rem; */
```
Comment (MarianaSouza) [inline styles/Home.module.css:44]: Pls remove the comment.

**Review by MarianaSouza — CHANGES_REQUESTED** [review]
There are just some details we could adapt to make this PR even better, @manecipoglu !

Please let me know if you have any questions:
https://www.loom.com/share/10c37afced934b9eaf6843d395c14964

**Comment by MarianaSouza** [comment]
Hey @ZahraB21 , I tagged you as a reviewer here too because of the video above I left for @manecipoglu . I think you could benefit from that info too <3

File: styles/Home.module.css [hunk 2]
```diff
@@ -30,55 +30,84 @@ hr.divider {
...
+  margin: 0 1rem;
+  font-size: 1.125rem;
   line-height: 1.5;
+  text-align: center;
+  font-weight: bold;
 }
 
+.button {
+  height: 2.4rem;
+  width: 25%;
```
Comment (MarianaSouza) [inline styles/Home.module.css:79]: Could you please change the width here to 8rem, @manecipoglu ?

**Review by MarianaSouza — CHANGES_REQUESTED** [review]
We're almost there, @manecipoglu ! Thank you!

**Review by MarianaSouza — APPROVED** [review]
Looks great! Thanks @manecipoglu  All yours to merge!

File: styles/Home.module.css [hunk 3]
```diff
@@ -30,55 +30,84 @@ hr.divider {
...
   line-height: 1.5;
+  text-align: center;
+  font-weight: bold;
 }
 
+.button {
+  height: 2.4rem;
+  width: 8rem;
+  border-radius: 5px;
+  background-color: #696969;
```
Comment (Schlickmann) [inline styles/Home.module.css:81]: [not a blocker] Just leaving this suggestion here... can we start using variables to hold these values, like colours and padding and margin...

e.g. `margin-base; margin-2; padding-base; padding-2; $colour-gray` and so on...

**Review by Schlickmann — APPROVED** [review]

File: styles/Home.module.css (hunk 3 above)
Comment (MarianaSouza) [reply styles/Home.module.css:81]: I love this idea for the next task, @Schlickmann ! Would you like to write a detailed issue to address that?

**Comment by manecipoglu** [comment]
I have been searching for the safe git commands to merge this for a while until I came to this page and see that I can do that with a single click of a button. GUI rocks :P

**Comment by MarianaSouza** [comment]
> I have been searching for the safe git commands to merge this for a while until I came to this page and see that I can do that with a single click of a button. GUI rocks :P

Right? Another good practice @manecipoglu is deleting the branch after merging a PR, which helps to keep the repo clean. You just need to click the "Delete branch: button above :)

**Comment by manecipoglu** [comment]
> > I have been searching for the safe git commands to merge this for a while until I came to this page and see that I can do that with a single click of a button. GUI rocks :P
> 
> Right? Another good practice @manecipoglu is deleting the branch after merging a PR, which helps to keep the repo clean. You just need to click the "Delete branch" button below :)

Magic :))

**Pull request merged**